aptos-move/framework/aptos-framework/sources/Token.move

```diff
@@ -76,6 +76,13 @@ module AptosFramework::Token {
     struct CreateTokenEvent has drop, store {
         id: TokenId,
         token_data: TokenData,
+        initial_balance: u64,
+    }
+
+    /// mint token event. This event triggered when creator adds more supply to existing token
+    struct MintTokenEvent has drop, store {
```

Comment:
Are you planning to emit the event in mint() in this PR or a follow up one?

Reply:
there will be a follow up one for adding public facing mint function.  will emit the events there

Reply:
Makes sense 👍